moor/lib/src/web/storage.dart

```diff
@@ -39,20 +39,32 @@ abstract class MoorWebStorage {
   /// However, older browsers might not support IndexedDB.
   @experimental
   factory MoorWebStorage.indexedDb(String name,
-      {bool migrateFromLocalStorage}) = _IndexedDbStorage;
+      {bool migrateFromLocalStorage, bool inWebWorker}) = _IndexedDbStorage;
```

Comment:
Let's add some documentation on what `inWebWorker` is doing here. Maybe something like: "When [inWebWorker] is set (defaults to false), the implementation will use [WorkerGlobalScope] instead of [window]".